docs/download_images.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python3
+
+#
+# Downloads all images referenced from the documentation
+# and replaces corresponding references within rst files.
+# NB: this script updates existing rst files!
+# do not commit such changes.
+#
```

Comment:
i'm tempted (and have considered this for a while) to write an rst viewer callback, if the images are stored locally at the users request anyway.. :)